codegen/protocol-test-codegen/build.gradle.kts

```diff
@@ -20,7 +20,7 @@ plugins {
 }
 
 dependencies {
-    implementation("software.amazon.smithy:smithy-aws-protocol-tests:0.9.8")
+    implementation("software.amazon.smithy:smithy-aws-protocol-tests:0.9.9")
```

Comment:
Do you know where does the original version number come from? Is this necessary?

Reply:
This is required, as it looks like the protocol_tests were run on smithy from master (which was on unpublished `0.9.9`)
When I ran protocol_tests using version `0.9.8`, command NullAndEmptyHeaders protocol tests added in https://github.com/awslabs/smithy/pull/349 were removed